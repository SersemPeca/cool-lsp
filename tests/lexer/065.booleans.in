False
FAlse
FaLse
FalSe
FalsE
FALse
FaLSe
FalSE
FAlSe
FaLsE
FAlsE
FALSe
FaLSE
FALsE
FAlSE
FALSE
