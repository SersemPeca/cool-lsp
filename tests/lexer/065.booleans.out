#1 TYPEID False
#2 TYPEID FAlse
#3 TYPEID FaLse
#4 TYPEID FalSe
#5 TYPEID FalsE
#6 TYPEID FALse
#7 TYPEID FaLSe
#8 TYPEID FalSE
#9 TYPEID FAlSe
#10 TYPEID FaLsE
#11 TYPEID FAlsE
#12 TYPEID FALSe
#13 TYPEID FaLSE
#14 TYPEID FALsE
#15 TYPEID FAlSE
#16 TYPEID FALSE
